### ISSUE TITLE
Add plugin injection support

### ISSUE THREAD

**Issue by petrpatek** [apify/fingerprint-suite#32]
We should also inject the collected plugin data for supported browsers.

**Comment by barjin** [comment]
I see that the plugins data (along with `mime-types`) are being already generated in the `fingerprint-generator`. Injecting those will probably not be any different from all the other properties. Is this somehow connected to #27 ? Thanks!

**Comment by petrpatek** [comment]
Yes, there is some weirdness with linking the plugins and mime-types together. I did a PR in the old repo. I am also afraid that injecting some no standard plugin could introduce way more uniqueness and actually the fingerprint could be easier to track and also won't blend with the site traffic. This is one of the reasons why we need to have tests for the performance of the fingerprint on the most popular protections.
